Documentation~/Cinemachine3rdPersonFollow.md

```diff
@@ -0,0 +1,75 @@
+# 3rd person follow
+
+Use Cinemachine Virtual Camera’s 3rd Person Follow to keep the camera at a constant position and distance relative to the Follow target (subject to damping controls), tracking the target’s movement and rotation.
+
+The virtual camera is bolted to a virtual rig that is attached to the target. The red rods and dots visible around the target in the scene view reflect the rig setup as well as the camera position and distance relative to the target.
+
+The rig is defined by three points: the **Shoulder Offset**, the **Vertical Arm Length**, and the **Camera Distance**.
+
+Thus, the camera position is completely controlled by the Follow target’s position and orientation, and its orientation will always match that of the Follow target.
+
+
+
+For example, a rig set up with these values:
+
+![](images/CinemachineRigInspectorValuesExample.png)
```

Comment:
image is way too low-res - illegible

Reply:
The image was sharp and HD through the end product generated by DocTools. Regardless, I took new images to support better understanding of camera placement on the rig. Those images look sharp and HD.